patroni/api.py

```diff
@@ -38,6 +38,7 @@ def _write_status_code_only(self, status_code):
         self.log_request(status_code)
 
     def _write_response(self, status_code, body, content_type='text/html', headers=None):
+        # TODO: try-catch ConnectionResetError: [Errno 104] Connection reset by peer and log it in DEBUG level
```

Comment:
Why do you implement neither of the options [you mentioned in the issue](https://github.com/zalando/patroni/issues/2021#issuecomment-1184118414)? Was there any problem?

Reply:
Hi @hughcapet , peace by peace I suppose. incremental improvement, continuous integration :) The support for HEAD seamed like a trivial change good on it's own. Catching and logging the exceptions or sending extra headers will need some actual development, like a number of unit tests, integration tests, and more rigor in assessing the impact.